hashicorp/boundary-ui
# feat: 🎸 Add loading indicators for initial loads of resources

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-12025

## Description
Added loading indicators for the initial long loads coming from the client daemon. Originally, I was thinking I would have to poll and check the status of the daemon to know if we got any results back since normally the daemon returns cached results immediately but calling with `force_refresh` is blocking and waits enough time to get all the results back.

## Screenshots (if appropriate):

https://github.com/hashicorp/boundary-ui/assets/5783847/299bbbde-fc13-4c4f-bc5f-55e7d6279560


## How to Test
Use a binary with a client daemon and add at least 10k targets. You should be able to see a load time that's noticeable now. 

## Checklist:
